.changeset/proud-dots-fry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog-react': minor
+'@backstage/plugin-catalog': minor
+---
+
+Adds `EntityContextMenuItemBlueprint` to enable extending the entity page's context menu
```

Comment:
As we do not have a documentation for this yet, could you please add usage examples here?

Reply:
Added!